varya/README.md

```diff
@@ -0,0 +1,127 @@
+# Varya
+
+Varya is a styling system for quickly creating Gutenberg-ready themes for WordPress.
```

Comment:
Should this be rephrased to make clear that it's a parent theme itself?

Reply:
Yup, will do!